deploy/journeyEngineStepFunction.asl.json

```diff
@@ -74,7 +74,7 @@
         "ipAddress.$": "$$.Execution.Input.ipAddress",
         "clientOAuthSessionId.$": "$$.Execution.Input.clientOAuthSessionId",
         "featureSet.$": "$$.Execution.Input.featureSet",
-        "isUserInitiated.$": "$.lambdaInput.isUserInitiated"
+        "lambdaInput.$": "$.lambdaInput"
```

Comment:
I think this is a sensible change :)

We could also consider matching the way we pass scope/context to the CRI states (i.e. using query params in the journey string), but there's not particular need.